cob_bringup/robots/cob4-4.xml

```diff
@@ -26,12 +26,13 @@
 			</include>
 			<include file="$(find cob_bringup)/drivers/bms.launch">
 				<arg name="robot" value="$(arg robot)"/>
+                <arg name="sim" value="$(arg sim)"/>
 			</include>
```

Comment:
also, since bms.launch now has the `flag`, it can be moved out of the `<group unless="$(arg sim)">`